io.mrarm.mcpelauncher-ui.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "io.mrarm.mcpelauncher-ui",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.14",
+    "command": "mcpelauncher-ui-qt",
+    "build-options" : {
```

Comment:
SDK has its own CFLAGS set, it's better to use it instead.

Reply:
Ok, the examples of flatpak had set them for whatever reasons.
Only the x86_64 gamelauncher don't compile with the default cflags of the sdk at all, clang-9, throws errors